cli/azd/pkg/input/external_prompt_client.go

```diff
@@ -18,38 +18,47 @@ type externalPromptClient struct {
 	pipeline httputil.HttpClient
 }
 
-type promptOptions struct {
-	Type    string               `json:"type"`
-	Options promptOptionsOptions `json:"options"`
+type externalPrompt struct {
+	ID      string                `json:"id"`
+	Type    string                `json:"type"`
+	Options externalPromptOptions `json:"options"`
 }
 
-type promptChoice struct {
+type externalPromptChoice struct {
 	Value  string  `json:"value"`
 	Detail *string `json:"detail,omitempty"`
 }
 
-type promptOptionsOptions struct {
-	Message      string          `json:"message"`
-	Help         string          `json:"help"`
-	Choices      *[]promptChoice `json:"choices,omitempty"`
-	DefaultValue *any            `json:"defaultValue,omitempty"`
+type externalPromptOptions struct {
+	Message      string                  `json:"message"`
+	Help         string                  `json:"help"`
+	Choices      *[]externalPromptChoice `json:"choices,omitempty"`
+	DefaultValue *any                    `json:"defaultValue,omitempty"`
 }
 
-type promptResponse struct {
+type exteranlPromptResponse struct {
```

Comment:
Nit
```suggestion
type externalPromptResponse struct {
```

Reply:
Thanks!  I spell it wrong once and IntelliSense and Copilot keep it around forever :-)